packages/node-opcua-client/test/test_client_alarm.ts

```diff
@@ -77,3 +90,55 @@ describe("Testing client alarm", () => {
         alarmList.length.should.eql(2);
     });
 });
+
+describe("Ensure that all fields in the ClientSessionPriv interface are valid", () => {
```

Comment:
I am not sure the test is relevant here.
It doesn't highlight the faulty behavior we want to fix and only test a trivial internal function areClientSessionPrivFieldsValid

Can we provide a test that actually reproduce the bug that causes this PR to appear instead  ?